greeting/src/greeter.ts

```diff
@@ -0,0 +1,7 @@
+export class Greeter {
+  constructor(private greeting: string) {}
```

Comment:
Be sure to declare your properties above the constructor, and assign them inside of the constructor too: 

```
export class Greeter {
  greeting: string;

  constructor(greeting: string) {
    this.greeter = greeting;
  }
  greet(name: string): string {
    return `${this.greeting}, ${name}!`;
  }
}
```